mdslib/Makefile.in

```diff
@@ -14,6 +14,12 @@ MKDIR_P=@MKDIR_P@
 LIBS = @LIBS@ @GLOBUS_LIBS@ $(LIBSOCKET) @LIBRESOLV@ @LIBM@
 MDSLIB_LIBS=-lTdiShr -lTreeShr -lMdsShr
 
+# WARNING: gfortran 10 dangerous fixes ... actual fortran code must be reviewed asap
```

Comment:
so this file is basically unchanged.

Reply:
just a reminder because I doubt in m4 file we will remember it :-)

Reply:
You are right I'll remove it